packages/avatars/src/styled/StyledSRSpan.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'avatars.sr_element';
+
+export const StyledSRSpan = styled.span.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})`
+  position: absolute;
+  margin: -1px;
+  border: 0;
+  padding: 0;
+  width: 1px;
+  height: 1px;
+  clip: rect(1px, 1px, 1px, 1px);
+  clip-path: inset(50%);
+  overflow: hidden;
+  white-space: nowrap;
+`;
```

Comment:
I may have missed if/where we handle an `sr-only` element so I've included this in the meantime.

Reply:
I have some CodeSandbox evidence that adding `role="img"` along with `aria-label` to the `<div>` surrounding the SVG is going to yield preferable results. And I think it falls better in line with a11y [recommendations](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/img_role). Let's pair up on this, if necessary.

Reply:
I think this works better for the actual experience. I have updated.